e2e/LongHaul/service/Program.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using CommandLine;
+using Mash.Logging;
+using Mash.Logging.ApplicationInsights;
+using static Microsoft.Azure.Devices.LongHaul.Service.LoggingConstants;
+
+namespace Microsoft.Azure.Devices.LongHaul.Service
+{
+    internal class Program
+    {
+        private static readonly IDictionary<string, string> s_commonProperties = new Dictionary<string, string>();
+        private static Logger s_logger;
+        private static ApplicationInsightsLoggingProvider s_aiLoggingProvider;
+
+        private static async Task Main(string[] args)
+        {
+            s_commonProperties.Add(RunId, Guid.NewGuid().ToString());
```

Comment:
We need a property that this is the service client, and also one on the device app that it is device client.